inc/field/ldapselectfield.class.php

```diff
@@ -49,7 +49,7 @@ public function getDesignSpecializationField(): array {
       $label .= _n('LDAP directory', 'LDAP directories', 1);
       $label .= '</label>';
 
-      $ldap_values = json_decode(plugin_formcreator_decode($this->question->fields['values']), JSON_OBJECT_AS_ARRAY);
```

Comment:
plugin_formcreator_encode() is not used in method PrepareQuestionInputForSave() then we should be able to remove plugin_formcreator_decode()